mypyc/emitfunc.py

```diff
@@ -189,7 +189,10 @@ def visit_set_attr(self, op: SetAttr) -> None:
 
     def visit_load_static(self, op: LoadStatic) -> None:
         dest = self.reg(op)
-        self.emit_line('%s = %s;' % (dest, op.identifier))
+        if is_int_rprimitive(op.type):
```

Comment:
I think this is cleaner, because it might be confusing to have code containing `CPyTagged_FromObject` stored in the identifier. By doing it the current way, we can look at the op type and just insert the `CPyTagged_FromObject` when we emit code.

Either way, I don't think it should matter too much.